resources.md

```diff
@@ -21,9 +21,9 @@ show_hero: true
 * [Enterprise support](#support)
 * [Managed services](#managed)
 * [Cloud](#cloud)
-  * [AWS](#aws)
-  * [Azure](#azure)
-  * [GCP](#gcp)
+  * [Amazon Web Services](#aws)
+  * [Microsoft Azure](#azure)
+  * [Google Cloud](#gcp)
 
```

Comment:
The list used to be alphabetical so that we didn't betray favoritism. To preserve that with the new wording, lines 25 and 26 need to swap places.

Reply:
Yes.. lets fix that @Jessie212

Reply:
Fixed.